src/client/Client.cc

```diff
@@ -15299,7 +15290,7 @@ void Client::handle_conf_change(const ConfigProxy& conf,
 
 void intrusive_ptr_add_ref(Inode *in)
 {
-  in->get();
+  in->iget();
 }
```

Comment:
You can delete this methods now. That's half the benefit of inheriting from `RefCountedObject` :)

Reply:
I change the `in->get()` --> `in->iget()` on purpose, just want to make it in pair with the `in->iput()`, which could make the code be easier to read. I just want to not muddle the `get()` from `RefCountedObject` with the one from the intrusive Inode ptr.

Reply:
Yeah, if you still think this is ugly, I can delete this :-)